multinode/console/controllers/nodes.go

```diff
@@ -73,6 +73,7 @@ func (controller *Nodes) Add(w http.ResponseWriter, r *http.Request) {
 		case nodes.ErrNodeAPIKeyInvalid.Has(err):
 			controller.serveError(w, http.StatusUnauthorized, ErrNodes.Wrap(err))
 		case nodes.Error.Has(err):
```

Comment:
I looked at the code and, in my opinion, this case is like the default one, we should return `InternalServerError`.

What's the reason you used `BadGateway`?

Reply:
@ifraixedes 
 `"error": "nodes web api controller: nodes: rpc: tcp connector failed: rpc: dial tcp: lookup storagenode4 on 127.0.0.53:53: server misbehaving"`
 The error was connection error so I used bad gateway ok I am changing it to `InternalServerError`